packages/node/src/yargs.ts

```diff
@@ -63,6 +63,23 @@ export const yargsOptions = yargs(hideBin(process.argv))
       return reindexInit(argv.targetHeight);
     },
   })
+  .command({
```

Comment:
I think we want to specify the args for this.

Reply:
It seems there are still other args missing. Db schema and project path are ones i can think of. Rather than redefining them for subcommands we should move them to a common place